src/control/SConscript

```diff
@@ -55,19 +55,20 @@ def install_go_bin(denv, gosrc, gopath, libs, name, install_name):
     # Go path to DAOS code
     path = join(gopath, "src", repopath, "src")
 
-    bin = get_bin_path(gopath, name)
+    bin_path = get_bin_path(gopath, name)
     mod_src = join(gosrc, name, "main.go") # Module src
     install_src = get_install_src_dir(repopath, name)
     install_bin = get_install_bin_path(denv, install_name)
     if libs is not None:
         src = [mod_src]
         src.extend(libs)
     else:
-        src = mod_src
-    denv.Command(bin, src,
+        src = [mod_src]
```

Comment:
I initially made it not a list in this case because that's how it was implemented before. If it works either way, we could simplify by putting the src = [mod_src] line before the if statement.

Reply:
fixed